src/index.css

```diff
@@ -1,35 +1,48 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700');
+
+html { 
+  background: url(assets/recycle-bg-edit.jpg) no-repeat center center fixed; 
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
 body {
   margin: 0;
   padding: 0;
-  font-family: sans-serif;
-  background: #eeeeee;
+  font-family: 'Open Sans', sans-serif;
+}
+#root {
+  background: none;
 }
-
 .centerStuff {
   text-align: center;
-  margin-top: 30vh;
+  margin-top: 35vh;
 }
 .mainLabel{
-  font-size: 25px;
-  color: #666666;
+  font-size: 37px;
+  color: #ffffff;
+  font-weight: 300;
+  margin-top: 10px;
 }
 .mainInput {
   margin-left: 20px;
-  height: 30px;
-  width: 300px;
+  height: 45px;
+  width: 500px;
+  border-radius: 10px;
+  background:rgba(255, 255, 255, 0.8);
 }
 .redButton{
-  background-color: red;
+  background-color: #EF8232;
   color: white;
-  font-size: 35px; 
+  font-size: 25px; 
```

Comment:
We should use `em` instead of `px` 

https://stackoverflow.com/questions/609517/why-em-instead-of-px